### PR TITLE
Use a standard window for the About dialog

### PR DESCRIPTION
**Pull request by toehead2001**
about

### CODE REVIEW

**Comment by tony971** [comment]
About Windows 10

Looks great on Windows 10.

**Comment by cxd4** [comment]
Any thoughts @exhalatio (or zilmar) ?  Does this help solve the issue you were thinking of?

Maybe other improvements missing to add on to this if it got merged?

File: Source/Project64/User Interface/Gui Class.cpp [hunk 1]
```diff
@@ -1019,24 +983,18 @@ DWORD CALLBACK AboutBoxProc(HWND hWnd, DWORD uMsg, DWORD wParam, DWORD lParam)
...
 	case WM_INITDIALOG:
 	{
-		enum { ROUND_EDGE = 15 };
-
-		DWORD dwStyle = GetWindowLong(hWnd, GWL_STYLE);
-		dwStyle &= ~(WS_CAPTION | WS_SIZEBOX);
-		SetWindowLong(hWnd, GWL_STYLE, dwStyle);
+		//Title
+		LONG_PTR originalWndProc = GetWindowLongPtrW(hWnd, GWLP_WNDPROC);
+		SetWindowLongPtrW(hWnd, GWLP_WNDPROC, (LONG_PTR)DefWindowProcW);
```
Comment (project64) [inline Source/Project64/User Interface/Gui Class.cpp]: you should not set the wndproc create the dialog with the W version

**Comment by MELERIX** [comment]
> Looks great on Windows 10.

that is because Windows 10 RTM does not have colours in the title bar and the borders of windows, but now in Windows 10 Insider Preview they have added colours for the title bar and the borders of windows.

File: Source/Project64/User Interface/Gui Class.cpp (hunk 1 above)
Comment (toehead2001) [inline Source/Project64/User Interface/Gui Class.cpp]: You mean change it to this?

`LONG_PTR originalWndProc = GetWindowLongPtr(hWnd, GWLP_WNDPROC);`
`SetWindowLongPtr(hWnd, GWLP_WNDPROC, (LONG_PTR)DefWindowProc);`

Comment (project64) [inline Source/Project64/User Interface/Gui Class.cpp]: I mean in CMainGui::AboutBox
it should be

DialogBoxParamW(GetModuleHandle(NULL), MAKEINTRESOURCEW(IDD_About), m_hMainWindow, (DLGPROC)AboutBoxProc,(LPARAM)this);

Comment (toehead2001) [inline Source/Project64/User Interface/Gui Class.cpp]: oh... That makes so much more sense. Lack of grammar can be so ambiguous :blush:

**Comment by toehead2001** [comment]
Does the pull look ok now?

File: Source/Project64/User Interface/Gui Class.cpp (hunk 1 above)
Comment (cxd4) [inline Source/Project64/User Interface/Gui Class.cpp]: glad I'm not the only one bad at reading his grammar, lol

**Comment by project64** [comment]
looks fine

**Pull request merged**